### ISSUE TITLE
Maps removed from maps/

### ISSUE THREAD

**Issue by lior8** [nathansttt/hog2#83]
PR #80 removed most of the maps from maps/. 
While they should be brought back, perhaps instead I can make a sh script which will simply download them instead, which will make the repo much more lightweight.

**Comment by nathansttt** [comment]
Yes! I realized this happened the other day, but haven't had a chance to fix it yet.

**Comment by lior8** [comment]
Following our conversation, I think it will be better to have a separate repo for that and then only include a bash script in hog2 which will download things selectively. It will also add the ability to download benchmarks from other repos (e.g., Iron Harvest).

**Comment by nathansttt** [comment]
I've restored the files for the moment. The plan is to create a benchmark director which is blank except for a script which will download from other repositories. I will leave this open until that is complete.
